dashboard/test/controllers/api/v1/pd/application/facilitator_applications_controller_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'test_helper'
+
+module Api::V1::Pd::Application
+  class FacilitatorApplicationsControllerTest < ::ActionController::TestCase
+    setup_all do
+      @test_params = {
+        form_data: build(:pd_facilitator1819_application_hash)
+      }
+    end
+
+    test_redirect_to_sign_in_for :create
+    test_user_gets_response_for :create, user: :student, params: -> {@test_params}, response: :forbidden
+    test_user_gets_response_for :create, user: :teacher, params: -> {@test_params}, response: :success
```

Comment:
Do we want to open access to this just yet?

Reply:
Good question. For that matter, it's already open https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/config/routes.rb#L417 but the url hasn't been shared yet. We do want to be able to test it.

Maybe we should either add a CDO variable in the meantime to enable this so we can test on adhoc and dev, or an environment check? @tanyaparker thoughts?

Reply:
Can we restrict to just `workshop_admin` permission until we go live?

Reply:
Yes, but then testing & bashing will be more cumbersome because we'd first need to create workshop_admins. Why don't I just temporarily block it on the production environment?

Reply:
Ohh. Yes, that sounds fine!